Assets/00_Content/Scripts/Vikings/States/BrawlingVikingState.cs

```diff
@@ -107,12 +117,15 @@ public class BrawlingVikingState : VikingState {
 
 		private VikingState DoPlayerBrawl() {
 
-			if ((navMeshAgent.transform.position - playerTarget.transform.position).sqrMagnitude < viking.Data.attackTriggerDistance && !viking.IsAttacking) {
+			if ((navMeshAgent.transform.position - playerTarget.transform.position).sqrMagnitude < Mathf.Pow(viking.Data.attackTriggerDistance, 2) && !viking.IsAttacking) {
```

Comment:
The `Pow` function is quite expensive, for such a trivial case you should just do: `x * x`.

Reply:
ok :)